### DISCUSSION TITLE
Callback embed Vimeo avec tac en mode automatique

### DISCUSSION THREAD

**Discussion by tromeurf** [AmauriC/tarteaucitron.js#1096] [General]
Bonjour, 
Afin d'utiliser un player vidéo custom (Plyr) pour lire mes embed vimeo, je cherche à déclencher une fonction quand les iframes vimeo sont insérée dans le DOM.
En mode automatique y a t'il un moyen de faire ça ?
Merci beaucoup pour votre aide, je tourne en rond et hésite à utiliser un MutationObserver.

**Comment by AmauriC** [comment]
Bonjour,

L'event "loaded" n'est pas suffisant ?

```
document.addEventListener("vimeo_loaded", function(){
   setTimeout(function() {
      // chargement de Plyr
   }, 500);
});
```

**Reply by tromeurf** [reply]
Si, c'est parfait, je ne connaissais pas l'existence des events de load.
Merci beaucoup
